docs/src/man/reshaping_and_pivoting.md

```diff
@@ -244,6 +244,23 @@ julia> first(widedf, 6)
 │ 6   │ setosa        │ 6     │ 1.7         │ 0.4        │ 5.4         │ 3.9        │
 ```
 
+You can even skip the `:variable` and `:value` variables and use:
```

Comment:
can you please add a comment what happens then (it is rewritten as `unstack(::DataFrame, :variable, :value)`)

Reply:
Not sure what you meant, but I showed the output in the following REPL result. Choosing :variable and :value as default variables is obvious to me but please feel free to add more description if you want.